src/admin.js

```diff
@@ -143,22 +147,45 @@ export class StitchAdminClient extends StitchClient {
               return {
                 get: ()=> api._get(valueUrl),
                 remove: ()=> api._delete(valueUrl),
-                update: (data) => api._put(valueUrl, data)
+                update: (data) => api._put(valueUrl, { body: JSON.stringify(data) })
               };
             }
           }),
 
+          hosting: () => ({
+            config: () => ({
```

Comment:
The ticket specifies these, but they're not implemented yet in the backend. Should I just remove them for now?

Reply:
you can leave em in for now. it'll just return a 404 error until it's used against a version of the server that has them implemented, but that's OK.